### PR TITLE
Added 12/01 Upcoming Releases Update

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

**Review by r0mant — APPROVED** [review]

**Review by alexfornuto — APPROVED** [review]

File: docs/pages/preview/upcoming-releases.mdx [hunk 1]
```diff
@@ -8,29 +8,30 @@ The Teleport team delivers a new major release roughly every 3 months.
 
 ## Teleport
 
-### Teleport 11.1
+### 11.2
+
+Teleport 11.2 will bring support for script-based automatic upgrades and
+improved Database joining experience in the Web UI.
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:14]:
```suggestion
an improved database joining experience in the Web UI.
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 2]
```diff
@@ -8,29 +8,30 @@ The Teleport team delivers a new major release roughly every 3 months.
...
-- FIPS support to Desktop Access
 - Improved x509 integration for self-hosted Microsoft SQL Server
-- ARM64 support for container images and CentOS 7
-- Auth connectors support in Kubernetes Operator
-- Advanced search for Teleport Connect
-- Improved Web UI performance when connecting to resources
-- Access Request Plugin for Discord
+- ARM64 support for CentOS 7
+- Signed macOS tar archives
+- Gracefully handle expired database certificates in Teleport Connect
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:19]:
```suggestion
- Graceful handling of database certificates in Teleport Connect
```

File: docs/pages/preview/upcoming-releases.mdx [hunk 3]
```diff
@@ -8,29 +8,30 @@ The Teleport team delivers a new major release roughly every 3 months.
...
+- Fixed per-session MFA issues
+- Improved database joining flow in Web UI
```
Comment (ptgott) [inline docs/pages/preview/upcoming-releases.mdx:21]:
```suggestion
- Improved database joining flow in the Web UI
```

**Review by ptgott — COMMENTED** [review]
Some small grammatical nits

**Review by zmb3 — APPROVED** [review]

**Pull request merged**